src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/ReblockGVCF.java

```diff
@@ -0,0 +1,437 @@
+import org.broadinstitute.hellbender.tools.walkers.haplotypecaller.HaplotypeCallerGenotypingEngine;
+import org.broadinstitute.hellbender.tools.walkers.haplotypecaller.ReferenceConfidenceMode;
+import org.broadinstitute.hellbender.utils.MathUtils;
+import org.broadinstitute.hellbender.utils.genotyper.IndexedSampleList;
+import org.broadinstitute.hellbender.utils.genotyper.SampleList;
+import org.broadinstitute.hellbender.utils.variant.GATKVCFConstants;
+import org.broadinstitute.hellbender.utils.variant.GATKVCFHeaderLines;
+import org.broadinstitute.hellbender.utils.variant.GATKVariantContextUtils;
+import org.broadinstitute.hellbender.utils.variant.writers.GVCFWriter;
+import picard.cmdline.programgroups.OtherProgramGroup;
+
+import java.io.File;
+import java.util.*;
+
+/**
+ * Perform joint genotyping on gVCF files produced by HaplotypeCaller
+ *
+ * <p>
+ * ReblockGVCF compressed a GVCF by merging hom-ref blocks that were produced using the '-ERC GVCF' or '-ERC BP_RESOLUTION' mode of the
+ * HaplotypeCaller according to new GQ band parameters.  A joint callset produced with GVCFs reprocessed by ReblockGVCF will have
+ * lower precision for hom-ref genotype qualities at variant sites, but the input data footprint can be greatly reduced
+ * if the default GQ band parameters are used.</p>
+ *
+ * <h3>Input</h3>
+ * <p>
+ * A HaplotypeCaller-produced gVCF to reblock
+ * </p>
+ *
+ * <h3>Output</h3>
+ * <p>
+ * A smaller GVCF.
+ * </p>
+ *
+ * <h3>Usage example</h3>
+ * <pre>
+ * gatk ReblockGVCF \
+ *   -R reference.fasta \
+ *   --variant sample1.g.vcf \
+ *   -O sample1.reblocked.g.vcf
+ * </pre>
+ *
+ * <h3>Caveats</h3>
+ * <p>Only single-sample gVCF files produced by HaplotypeCaller can be used as input for this tool.</p>
+ * <p>By default this tool only passes through annotations used by VQSR.  A different set of annotations can be specified with the usual -A argument.  Annotation groups are not supported by this tool.</p>
+ *
+ * <h3>Special note on ploidy</h3>
+ * <p>This tool assumes diploid genotypes.</p>
+ *
+ */
+@ExperimentalFeature
+@CommandLineProgramProperties(summary = "Compress a single-sample GVCF from HaplotypeCaller by merging homRef blocks using new GQ band parameters",
+        oneLineSummary = "Condenses homRef blocks in a single-sample GVCF",
+        programGroup = OtherProgramGroup.class,
+        omitFromCommandLine = true)
+@DocumentedFeature
+public final class ReblockGVCF extends VariantWalker {
+
+    private final static int PLOIDY_TWO = 2;  //assume diploid genotypes
+
+    @Argument(fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME, shortName = StandardArgumentDefinitions.OUTPUT_SHORT_NAME,
+            doc="File to which variants should be written")
+    private File outputFile;
+
+    @Argument(fullName=GenotypeGVCFs.ALL_SITES_LONG_NAME, shortName=GenotypeGVCFs.ALL_SITES_SHORT_NAME, doc="Include loci found to be non-variant after genotyping")
```

Comment:
It's a little weird to make a boolean argument default to true.  Should the name be inverted so that you specify it to disable it?